modules/data.land/R/InventoryGrowthFusion.R

```diff
@@ -114,27 +126,35 @@ model{
     ## Substitute into code
     TreeDataFusionMV <- sub(pattern = "## RANDOM EFFECT TAUS", Rpriors, TreeDataFusionMV)
     TreeDataFusionMV <- gsub(pattern = "## RANDOM_EFFECTS", Reffects, TreeDataFusionMV)
-  }
+  }   ### END RANDOM EFFECTS
 
+  ########################################################################
+  ###
+  ###        FIXED EFFECTS
+  ###
+  ########################################################################
   if(FALSE){
     ## DEV TESTING FOR X, polynomial X, and X interactions
-    fixed <- "X + X^3 + X*bob + bob + dia"
+    fixed <- "X + X^3 + X*bob + bob + dia + X*Tmin[t]" ## faux model, just for testing jags code
   }
```

Comment:
It would be nice to pull things like this out as formal unit tests. When I develop, I usually try to write tests immediately after, or even during or before, the rest of my code, and then use the `devtools::load_all(...)` to reload all of the code in a package and run the tests there.